README.md

```diff
@@ -8,21 +8,38 @@
 [![license](https://img.shields.io/github/license/Skyscanner/backpack-android.svg)](https://github.com/Skyscanner/backpack-android)
 [![platform](https://img.shields.io/badge/platform-android-green.svg)](https://github.com/Skyscanner/backpack-android)
 
-## Installation
+Backpack for Android supports two targets: **Android View** system and **Jetpack Compose**.
 
-Backpack is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-android). To install
-all of it, add the following line to your `build.gradle` (in your app module) in the `dependencies` block:
+Backpack is available through Maven Central. Thus, before adding Backpack to your project,
+make sure Maven Central is in your repositories list. Add this to your root `build.gradle`:
 
 ```gradle
-implementation 'net.skyscanner.backpack:backpack-android:33.2.0'
+repositories {
```

Comment:
This is common part of Android and Compose, so it's extracted

Reply:
Maybe we can have a common installation part which has both `backpack-android` and `backpack-compose` defined (as in two lines for both - maybe a comment specifying what's what). Having the installation parts split across two sections feels a bit disconnected - I generally want to know: how can i add this thing to my app. and then: how do i use this.